python/ray/tests/test_failure.py

```diff
@@ -125,15 +125,23 @@ def expect_exception(objects, exception):
     ray.get(signal2.send.remote())
 
 
-@pytest.mark.skipif(True, reason="run_function_on_all_workers doesn't work")
 def test_failed_function_to_run(ray_start_2_cpus, error_pubsub):
     p = error_pubsub
 
     def f(worker):
         if ray._private.worker.global_worker.mode == ray.WORKER_MODE:
             raise Exception("Function to run failed.")
 
+    @ray.remote
+    class Actor:
+        def foo(self):
+            pass
+
     ray._private.worker.global_worker.run_function_on_all_workers(f)
+
+    actors = [Actor.remote() for _ in range(2)]
+    ray.get([actor.foo.remote() for actor in actors])
```

Comment:
nit: add comment explaining that actor is necessary to assign jobid to each worker.

Also why does the actor class definition have to go before `run_function_on_all_workers`?